test/commands/login/index.test.ts

```diff
@@ -132,9 +143,23 @@ describe('login command', () => {
           .it("chains the show project and doesn't throw an error", (ctx) => {
             expect(ctx.stdout).to.not.contain(notFoundProject)
           })
+
+        it('checks that the config contains some data', () => {
+          const config = configService.show()
+          expect(config.currentUserId).to.equal(testUserId)
+          expect(config.version).to.equal(getMajorVersion())
+          expect(config.configs).to.haveOwnProperty(testUserId)
+          expect(config.configs[testUserId].activeProjectId).to.equal(
+            projectSummary.project.projectId,
+          )
+          expect(config.configs[testUserId].outputFormat).to.equal('plaintext')
+        })
       })
 
-      describe('And When the user has several projects', () => {
+      describe.only('And When the user has several projects', () => {
```

Comment:
nitpick: lowercase "When" please

Reply:
but you just made me noticed that I've forgotten to remove the .only !!!!